Dockerfile.webhook

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.11 as builder
```

Comment:
Is it possible to make it compatible with docker versions that don't support build phases? (For one, this is the version that is shipped with centos 7.x.)

If interested, here is the patch that fixes the build for centos for me: https://github.com/booxter/multus-cni/commit/c5aed8d8a2ba58b69ad760e3c08de25f9859f22f

Reply:
done